docs/apis-tools/working-with-apis-tools.md

```diff
@@ -66,21 +66,21 @@ It is also possible to [build your own client](../apis-tools/build-your-own-clie
 
 ![Architecture diagram for Camunda Platform including all the components for SaaS](./img/ComponentsAndArchitecture_SaaS.png)
 
-<DocCardList items={[{type:"link", href:"/docs/apis-tools/tasklist-api/tasklist-api-overview/", label: "Tasklist API (GraphQL)", docId:"apis-tools/tasklist-api/tasklist-api-overview"},
+<DocCardList items={[{type:"link", href:"/docs/next/apis-tools/tasklist-api/tasklist-api-overview/", label: "Tasklist API (GraphQL)", docId:"apis-tools/tasklist-api/tasklist-api-overview"},
 {
-type:"link", href:"/docs/apis-tools/operate-api/", label: "Operate API (REST)", docId:"apis-tools/operate-api/operate-api-overview"
+type:"link", href:"/docs/next/apis-tools/operate-api/overview/", label: "Operate API (REST)", docId:"apis-tools/operate-api/operate-api-overview"
```

Comment:
@christinaausley I don't know why I had to add `overview` here but you didn't in your commit? I confirmed these links in my browser, without `overview` I was getting a 404. Do you know anything that would have caused this difference?